inyoka_theme_default/templates/errors/400_csrf.html

```diff
@@ -2,36 +2,60 @@
     errors/400_csrf.html
     ~~~~~~~~~~~~~~~~~~~~
 
-    Rendered on CSRF attempts.
+    Rendered on failed CSRF attempts.
 
     :copyright: (c) 2013-2016 by the Inyoka Team, see AUTHORS for more details.
     :license: BSD, see LICENSE for more details.
 #}
 
 {% extends 'base.html' %}
 {% block title %}
-  400 - {{ super() }}
+  {{ _('Forbidden (403)') }} - {{ super() }}
```

Comment:
Actually a 400 is more precise since the Request failed because it was incorrect and not because of missing permissions.